tests/api/fmr/samples/df/hierarchy_schema.json

```diff
@@ -388,34 +385,30 @@
                                 "id": "OPTION_TYPE",
                                 "localRepresentation": {
                                     "enumerationFormat": {
-                                        "textType": "String"
+                                        "dataType": "String"
                                     },
                                     "enumeration": "urn:sdmx:org.sdmx.infomodel.codelist.Codelist=BIS:CL_OPTION_TYPE(1.0)"
                                 },
                                 "conceptIdentity": "urn:sdmx:org.sdmx.infomodel.conceptscheme.Concept=BIS:CONCEPTS(1.0).OPTION_TYPE"
                             }
                         ],
-                        "timeDimensions": [
-                            {
-                                "position": 4,
-                                "type": "TimeDimension",
-                                "links": [
-                                    {
-                                        "rel": "self",
-                                        "type": "timedimension",
-                                        "uri": "https://raw.githubusercontent.com/sdmx-twg/sdmx-json/develop/structure-message/tools/schemas/2.0.0/sdmx-json-structure-schema.json",
-                                        "urn": "urn:sdmx:org.sdmx.infomodel.datastructure.TimeDimension=BIS.TEST(1.0).TIME_PERIOD"
-                                    }
-                                ],
-                                "id": "TIME_PERIOD",
-                                "localRepresentation": {
-                                    "format": {
-                                        "textType": "ObservationalTimePeriod"
-                                    }
-                                },
-                                "conceptIdentity": "urn:sdmx:org.sdmx.infomodel.conceptscheme.Concept=BIS:CONCEPTS(1.0).TIME_PERIOD"
-                            }
-                        ]
+                        "timeDimension": {
+                            "links": [
+                                {
+                                    "rel": "self",
+                                    "type": "timedimension",
```

Comment:
If I understand correctly, there is no `type` property in the definition of the `TimeDimension` object.

Reply:
Indeed. Same for the Dimension object. In addition, for timeDimension, `position` is not allowed

Reply:
Hopefully, when we get stricter json schemas, we may be able to resolve such cases :-)